crates/extensions/tedge_mqtt_ext/src/lib.rs

```diff
@@ -197,6 +197,30 @@ impl Actor for MqttActor {
             }
         };
 
+        let (tx, mut outgoing_messages_finished) = tokio::sync::mpsc::channel(1);
```

Comment:
They are outgoing messages in my mind because this bit concerns translating an internal actor message into an MQTT message, and the MQTT direction is the one that matters?